### PR TITLE
magicbell-cli 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by smeijer**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/magicbell-cli.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir[libexec/"bin/*"]
+  end
+
+  test do
+    assert_match(version.to_s, shell_output("#{bin}/magicbell --version 2>&1"))
+    assert_match(/you are not logged in/i, shell_output("#{bin}/magicbell logout 2>&1"))
```
Comment (chenrui333) [inline Formula/magicbell-cli.rb:24]:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/magicbell --version")
    assert_match /you are not logged in/i, shell_output("#{bin}/magicbell logout")
```

**Review by chenrui333 — CHANGES_REQUESTED** [review]
The artifact does not have LICENSE file

**Comment by chenrui333** [comment]
```
$ tree .
.
├── README.md
├── bin
│   └── magicbell.js
├── dist
│   └── index.cjs
└── package.json
```

**Comment by smeijer** [comment]
My bad. Fixed 🙂  

```
❯ tree .
.
├── LICENSE
├── README.md
├── bin
│   └── magicbell.js
├── dist
│   └── index.cjs
└── package.json

3 directories, 5 files
```

File: Formula/magicbell-cli.rb (hunk 1 above)
Comment (smeijer) [reply Formula/magicbell-cli.rb]: When I'd remove the braces, the second assert_match makes `brew audit` fail. Technically, I can remove the braces from the first line, but. having two different styles in a single block doesn't look pretty either.

I've removed the stderr redirect.

```
magicbell-cli
  * line 24, col 18: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.
```

**Comment by smeijer** [comment]
Not sure why one of the tests fails. Looks like hick-up in ci?

**Comment by chenrui333** [comment]
I am a legal expert, but reading thru the license made me hard to think this is a OSS license. Do mind checking https://spdx.org/licenses/ to see if there is anything similar to your license? Thanks!

**Comment by smeijer** [comment]
@chenrui333, i'll look into it and open a new PR when it's sorted. Thanks!

**Pull request closed without merging**